README.md

```diff
@@ -1,26 +1,96 @@
 # Template for NREL's Python packages
 
-This is an opinionated template for creating Python packages at NREL.
+This is an opinionated template for creating Python packages at NREL,
+intended for a wide range of users. If you are new to NREL or don't have
+much experience building Python packages, this template is a good option
+to provide you a scaffolding to get started quickly. It also sets up several
+support tools for you, such as automated checks on your code, automatic 
+documentation generation, and tests validation on every pull request.
```

Comment:
I think we need to add a description of all support tools that are included in this template somewhere (not for the Quickstart, though).

Reply:
I opened an issue #24 so we don't forget about that.